### PR TITLE
eks-update-global-values

### PR DESCRIPTION
**Pull request by calvix**
update values to match new scheme https://github.com/giantswarm/cluster-eks/pull/50

### CODE REVIEW

**Comment by calvix** [comment]
EKS test passed, not sure what is the rest but this changes only values for EKS so it should not really matter for the rest.

and its kinda needed to merge to avoid errors for e2e for ESK as the chart already has the new scheme

File: providers/eks/upgrade/test_data/cluster_values.yaml [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-    instanceType: r6i.xlarge
-    maxSize: 3
-    minSize: 3
-    rootVolumeSizeGB: 25
+  # We need to pass the node pools otherwise the test called "has all the worker nodes running" will fail because it
+  # expects to find the number of worker nodes in the helm values, but the value is not there if don't pass it, as it's
+  # defaulted in the chart template.
+  # @TODO: https://github.com/giantswarm/giantswarm/issues/28063
+  nodePools:
+    nodepool-0:
```
Comment (fiunchinho) [inline providers/eks/upgrade/test_data/cluster_values.yaml:20]:
```suggestion
    # We are using a name with 10 chars which is the max number of characters allowed by our kyverno policies.
    nodepool-0:
```

File: providers/eks/standard/test_data/cluster_values.yaml [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-    instanceType: r6i.xlarge
-    maxSize: 3
-    minSize: 3
-    rootVolumeSizeGB: 25
+  # We need to pass the node pools otherwise the test called "has all the worker nodes running" will fail because it
+  # expects to find the number of worker nodes in the helm values, but the value is not there if don't pass it, as it's
+  # defaulted in the chart template.
+  # @TODO: https://github.com/giantswarm/giantswarm/issues/28063
+  nodePools:
+    nodepool-0:
```
Comment (fiunchinho) [inline providers/eks/standard/test_data/cluster_values.yaml:20]:
```suggestion
    # We are using a name with 10 chars which is the max number of characters allowed by our kyverno policies.
    nodepool-0:
```

**Comment by calvix** [comment]
not sure what is happening with all the test but the EKS test itself pass it fails at the end when its trying to remove the org, this was not the case yesterday

**Comment by calvix** [comment]
yay green!

**Review by njuettner — APPROVED** [review]

**Pull request merged**